src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -112,6 +148,16 @@ class EmojiPickerMenu extends Component {
         })();
     }
 
+    /**
+     * Return a unique key for each emoji item
+     *
+     * @param {Object} item
+     * @returns {String}
+     */
+    keyExtractor(item) {
+        return (`emoji_picker_${item.code}`);
+    }
```

Comment:
Can we move this to the utils method in `EmojiUtils` since it's also used in `EmojiPickerMenu/index.js`.

Reply:
If we want to make it more general, we could use `keyExtractorCode` and return only `'${item.code}'`. But honestly, I'm not sure if that's the best decision.
In this case, I would love just return `'${index}_${item.code}'` is it ok for you?

Reply:
Yup, I'm okay with that! 👍